__test__/DirectGift.test.js

```diff
@@ -0,0 +1,63 @@
+const { By, until, Key } = require('selenium-webdriver');
+const { driver } = require('./helper');
+const { load } = require('./Pages');
+
+describe('direct gift', () => {
+  beforeEach(async () => {
+    await load();
+  });
+  it('should direct gift', async () => {
+    const url = await driver.getCurrentUrl();
+    await driver.navigate().to(`${url}s/sagar-aryal`);
+    await driver.wait(until.elementLocated(By.className('donateButton'))).click();
+    await driver.switchTo().activeElement();
+    await driver.wait(until.elementLocated(By.id('treeDonateContinue')), 10000).click();
+    await driver.findElement(By.name('firstName')).sendKeys('Bright');
+    await driver.findElement(By.name('lastName')).sendKeys('Amidiagbe');
+    await driver.findElement(By.name('email')).sendKeys('captainamiedi1111@gmail.com');
+    await driver.findElement(By.name('address')).sendKeys('43 block');
+    await driver.findElement(By.name('city')).sendKeys('surulere');
```

Comment:
@captainamiedi are you fine with having your data here? We can also replace it with some anonymous data.